config/Config-build.in

```diff
@@ -184,6 +184,22 @@ menu "Global build settings"
 		  this per package by adding PKG_CHECK_FORMAT_SECURITY:=0 in the package
 		  Makefile.
 
+	config PKG_ASLR_PIE
+		bool
+		prompt "User space ASLR PIE compilation"
+		select BUSYBOX_DEFAULT_PIE
+		default y
```

Comment:
I would deactivate this by default for the beginning to see if it breaks too much.

Reply:
I think it's way too early to turn this on by default. This will probably break quite a few devices with smaller Flash/RAM sizes as well.